README.md

```diff
@@ -94,156 +94,146 @@ Example Job
 # API
 
 For any methods that needs `key` and `reference`
-you need to provide at least one of them but not necessarily both, keep in mind that reference override id.
+you need to provide at least one of them but not necessarily both, keep in mind that reference overrides id.
 ## Profile
 
 
-Retreive all profiles that match the query param, only source_ids are required
-
-```python
-client.profile.searching.list(source_keys=["source_key"], page=1, limit=30,.
-                             sort_by='created_at', order_by="desc",
-                             text_keywords=['python'], 
-                             created_at_min='2020-07-09T13:35:11+0000')
-```
-source_keys is required
-
-* Profile parsing:   
-Add a profile resume as binary to a given source
-
+### 📖 **The Profile Object**
+JSON object with 5 required fields: `key`, `reference`, `info`, `text_language` and `text`. The JSON contains other optional fields that cannot be listed here. (See the examples above)
+### 🧠 **Parse a Resume in a Source**
+This endpoint allows you to parse a resume and make a profile object from it.
+> 📘 **Real-time parsing**: To use the real-time parsing feature, you must have it enabled for the correponding source. In which case you just need to set `sync_parsing` to `1`.
+- Open the file in `binary mode`
 ```python
 with open('path/2/file', "rb") as f:
     profile_file = f.read()
-
-resp = client.profile.parsing.add_file(source_key="source_key", 
+```
+- Parse it using this method without reference:
+```python
+response = client.profile.parsing.add_file(source_key="source_key", 
                                   profile_file=profile_file, sync_parsing=1,
                                   sync_parsing_indexing=1,
                                   webhook_parsing_sending=0, 
                                   tags=[{"name":"archive", "value":True}])
 ```
 
-* Retrieve Parsing Object:     
+- Or using a reference like this:
+```python
+response = client.profile.parsing.add_file(source_key="source_key", reference = "my_resume",
+                                  profile_file=profile_file, sync_parsing=1,
+                                  sync_parsing_indexing=1,
+                                  webhook_parsing_sending=0, 
+                                  tags=[{"name":"archive", "value":True}])
+```
```

Comment:
You can present it like this : 

You can parse a resume bla bla like this : request without reference 

If you would like to specify a custom identifier that you can use later to retireve the parsed profile easily you can specify it using the argument reference : etc ...